Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -249,14 +249,23 @@
   categories:
     - UplinkUtility
 
-#TODO: Increase the price of this to 4-5/remove it when we get encrpytion keys
 - type: listing
   id: UplinkHeadset
-  name: Syndicate Overear-Headset
-  description: A headset that allows you to listen in on departmental channels, or contact other traitors.
+  name: Syndicate Headset - Overear-Headset
+  description: A headset that allows you to listen in on departmental channels, or contact other our agents. Should be flashband proof.
```

Comment:
Spelling, also don't change other descriptions in a pr like this especially when typos are being added.

Reply:
I don't know what spelling mistake is here, please specify.

Reply:
"flashban**d** proof"

Reply:
```suggestion
  description: A headset that allows you to listen in on departmental channels, or contact other our agents. Should be flashbang proof.
```